### PR TITLE
Add profile client

### PR DESCRIPTION
**Pull request by sam-by-name**
Last thing to sort its passing an object of a dynamic size back to the api, and to get an api going that handles this object

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,5 @@ bundle*
 *.log
 .DS_Store
 *.sqlite*
+server/db/dev.sqlite3
```
Comment (don-smith) [inline .gitignore]: remove

File: client/components/App.jsx [hunk 2]
```diff
@@ -28,6 +28,7 @@ const App = () => {
           <Route path='/login' component={Login} />
           <Switch>
             <Route path='/companies/add' component={CompanyAdd} />
+            <Route path='/companies/profile/:id' component={CompanyInfo} />
```
Comment (don-smith) [inline client/components/App.jsx]: remove

File: client/components/CompanyProfileEdit.jsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    const user = {
+      name: this.state.name,
+      count: this.state.count,
+      avgYearExp: this.state.avgYearExp,
+      femaleCount: this.state.femaleCount,
+      maleCount: this.state.maleCount,
+      openReqs: this.state.openReqs,
+      percentLocal: this.state.percentLocal
+    }
+    //Function to put into database goes here
```
Comment (don-smith) [inline client/components/CompanyProfileEdit.jsx]: `// Function to pass it to the API`

File: server/db/companies.js [hunk 4]
```diff
@@ -25,3 +27,12 @@ function addCompany (company, db = knex) {
...
     .then(companyId => companyId[0])
 }
+
+function getProfile (id, db = knex) {
+  return db('roles').where(id = 'company_id')
+}
+
+function addProfile (profile, db = knex) {
+  return db('role_counts')
+    .insert(profile)
```
Comment (don-smith) [inline server/db/companies.js]: probably need to convert from camelCase to train_case here

**Review by don-smith — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (sam-by-name) [reply .gitignore]: complete

File: client/components/App.jsx (hunk 2 above)
Comment (sam-by-name) [reply client/components/App.jsx]: complete

File: client/components/CompanyProfileEdit.jsx (hunk 3 above)
Comment (sam-by-name) [reply client/components/CompanyProfileEdit.jsx]: complete

File: server/db/companies.js (hunk 4 above)
Comment (sam-by-name) [reply server/db/companies.js]: I believe that's part of AddProfile branch

**Pull request merged**